vars/mavenExecuteStaticCodeChecks.groovy

```diff
@@ -9,6 +10,30 @@ import static com.sap.piper.Prerequisites.checkScript
 @Field String METADATA_FILE = 'metadata/mavenStaticCodeChecks.yaml'
 @Field String STEP_NAME = getClass().getName()
 
+
+@Field Set STEP_CONFIG_KEYS = [
+    /**Parameter to turn off SpotBugs.  */
+    'spotBugs',
+    /** Parameter to turn off PMD.*/
+    'pmd',
+    /** Maven modules which should be excluded by the static code checks. By default the modules 'unit-tests' and 'integration-tests' will be excluded.*/
+    'mavenModulesExcludes',
+    /** Path to a filter file with bug definitions which should be excluded.*/
+    'spotBugsExcludeFilterFile',
```

Comment:
Isn't that a duplicate to what you defined in the metadata.yml?

Reply:
yes but it has to go somehow to the config pages, or am I on a wrong path?

Reply:
I think this all can be removed. Se checkmarx example